lib/sqlalchemy/sql/selectable.py

```diff
@@ -4636,7 +4667,7 @@ def _normalize_froms(
         if froms:
             toremove = set(
                 itertools.chain.from_iterable(
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325